blog-meeting-minutes/2024-06-21-community-hour.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: community-office-hour-2024-06-21
+title: Community Office Hour 2024-06-21
+authors:
+    - matbmoser
+tags: [community, meeting-minutes]
+---
+
+## Office Hour meeting minutes
+
+### Infrastructure
+
+- Info from Test Management will be taken over by [Harald Zimmer](https://github.com/ds-hzimmer), please contact with your product owners to get a meeting with Harald and get the access if you are in the association.
+- Feedback requested from [Harald Zimmer](https://github.com/ds-hzimmer) for who is using: https://github.com/catenax-ng/product-test-data-generator
+- Branch protecting example provided by [Tom Meyer](https://github.com/tom-rm-meyer-ISST)
+
+### Security team
+
+- No security team will be available for R24.08, [Rohan Krish](https://github.com/RoKrish14) has told he will be going in July, contact him if you know who will be responsible for security in the Association.
+- Handover from [Rohan Krish](https://github.com/RoKrish14) will be done in this ticket: https://github.com/eclipse-tractusx/sig-security/issues/83
+
+### FOSS
+
+- Committer Election for [Lucas Capellino](https://github.com/arnoweiss), please vote if you are a committer!
+- [Evelyn Gurschler](https://github.com/evegufy) election to project lead concluded successfully, congratulations to our new project lead!
+- [Committer Election](https://projects.eclipse.org/projects/automotive.tractusx/elections/election-lucas-capellino-eclipse-tractus-x) for [Lucas Capellino](https://github.com/ds-lcapellino) open for voting again
+- Don't forget to update the legal docs regarding CC-BY-4.0 for documentation! Close the tickets in your repositories if its done: [eclipse-tractusx/sig-infra#477](https://github.com/eclipse-tractusx/sig-infra/issues/477)
```

Comment:
the change is mainly regarding CC-BY-4.0 but not only, I'd shorten to "Don't forget to update the legal docs!"

Reply:
ok